src/plugins/legacy_plugin.h

```diff
@@ -35,7 +37,7 @@ class LegacyPlugin
 	virtual ~LegacyPlugin();
 
 	// Called when a plugin DLL gets loaded.
-	virtual bool Init(char*);
+	virtual bool Init(TCHAR*);

```

Comment:
These (plugin vtable methods in the plugin ABI) can't be made TCHAR.  The plugin vtable is a fixed binary standard and the pre-existing versions must not change, for backwards compatibility.  (And the legacy plugin vtable support is only there exclusively for backwards compatibility; revising it to use wchar_t instead of char in the future would defeat the whole purpose of including the legacy plugin vtable logic at all.)

Reply:
An artifact from old code. TCHAR was a quick solution while testing Unicode flagging. I've been starting to build wchar_t's and wstrings throughout my code.